orchagent/fdborch.h

```diff
@@ -63,6 +58,12 @@ struct FdbData
     unsigned int vni;
 };
 
+struct FdbFlushUpdate
```

Comment:
I dont see a reason for this change. Please move it back.

Reply:
@prsunny : Thanks for the comment, must be due to some merge issue the code position got changed. 
Reverted back.